File: src/main/java/com/kate/Hello.java
package com.kate;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello world");
        Person p = new Person();
        p.hello();
        int age = 19;
        Integer age2 =19; //表示19是放在物件當中，即可以使用其功能 (八種資料型態皆有對應類別，第一個字母為大寫)
        char c = '我';
        byte b =120;
        float weight = 66.5f; //大小寫f都可以
        double height =1.7;
        boolean adult = true; //儲存真假使用布林值
        boolean enroll = false;
    }
}

File: src/main/kotlin/com/kate/Hello.kt
package com.kate

fun main(args: Array<String>) {
//    println("Hello kotlin")
//    Human().hello()
    val h =Human()
    h.hello()
    var age =19
    age = 20
    var weight = 66.5f //kotlin會自動判斷為double，如果希望資料型態為Float可以在最後打f
    var wei : Float =66.5f //或是左邊給資料型態
    var name : String //先定義變數名稱要先給資料型態
    name = "Kate"
}

class Human{
    fun hello() {
        println("Hello kotlin")
    }
}
